Platform/Teams/Tools and Improvements/[2019-06-11] State of E2E Testing on VA.gov.md

```diff
@@ -0,0 +1,53 @@
+# State of E2E Testing on VA.gov
```

Comment:
file location: maybe not at the top-level for this team? maybe make a "Notes" subfolder, particularly since this seems to be dated research rather than a living guide?